org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/symbols/LSPSymbolInWorkspaceHandler.java

```diff
@@ -56,10 +57,13 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 		if (site == null) {
 			return null;
 		}
-		final var dialog = new LSPSymbolInWorkspaceDialog(site.getShell(), project);
+		var styleProvider = new BoldStylerProvider(site.getShell().getFont());
+		final var dialog = new LSPSymbolInWorkspaceDialog(site.getShell(), project, styleProvider);
 		if (dialog.open() != IDialogConstants.OK_ID) {
```

Comment:
what about
```
int code = dialog.open();
styleProvider.dispose();
if (code != IDialogConstants.OK_ID) {
	return null;
}
```
then we do not need to repeat the `dispose` call in both branches

Reply:
See PR #803